media-video/guvcview/guvcview-2.2.1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Simple Qt6 or GTK+3 interface for capturing and viewing video from v4l2 devices"
+HOMEPAGE="https://guvcview.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/project/${PN}/source/${PN}-src-${PV}.tar.bz2"
+S="${WORKDIR}/${PN}-src-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl pulseaudio qt6 +sdl sfml"
+REQUIRED_USE="
+	|| ( sdl sfml )
```

Comment:
This might warrant a comment as it wasn't really clear to me how this related to the GUI situation.

Reply:
Alternative sdl/sfml for render engine. Command-line arg is allowed, but like GUI : 
sdl by default. SFML for alternative render x11-engine.
REQUIRED_USE removed